lib/subprocess.js

```diff
@@ -184,13 +179,15 @@ class SubProcess extends EventEmitter {
         });
       };
 
+      const encoding = this.opts.encoding || 'utf8';
+
       if (this.proc.stdout) {
-        this.proc.stdout.on('data', (chunk) => handleOutput({stdout: chunk.toString(), stderr: ''}));
+        this.proc.stdout.on('data', (chunk) => handleOutput({stdout: decode(chunk, encoding), stderr: ''}));
```

Comment:
should it only be applied to Windows?

Reply:
also, I assume it only makes sense to apply decode if the requested encoding differs from the default one (e.g. `utf8`)